pages/platform/data-processing/36_HOWTO_use_object_storage_java/guide.en-asia.md

```diff
@@ -0,0 +1,190 @@
+---
+title: How to use object storage with Apache Spark on the Data Processing platform using Java
+slug: object-storage-java
+excerpt: Object storage based on Swift and its S3 API is the common way to store data. Let’s find out how to do it in Java!
```

Comment:
Lookk at comment from 35_HOWTO and apply them here as well when it applies.
This comment applies to the whole file, not only this line.